imager/README.txt

```diff
@@ -1,33 +1,34 @@
 Raspberry Pi 4 EEPROM bootloader rescue image
 *********************************************
 
-The Raspberry Pi 4 has a small EEPROM used to store the bootloader.
+The Raspberry Pi 4 contains a small EEPROM used to store the bootloader.
 
 This rescue image reverts the bootloader EEPROM to factory default settings.
 
-This rescue image also updates the USB 3.0 (VL805) firmware to the latest
-version (138a1) with better full-speed Isochronous endpoint support.
+This rescue image also updates the USB 3 controller (VL805) firmware to the
+latest version, 138a1, which has better full-speed isochronous endpoint 
+support.
 
-This easiest method for creating EEPROM rescue images or formatting sdcards
-is to use the Raspberry Pi Imager from https://raspberrypi.com/software.
-The imager provides a GUI for downloading the latest version of this rescue
-image and flashing it to a spare SD CARD.
+The easiest method for creating EEPROM rescue images, and formatting SD 
+cards, is to use Raspberry Pi Imager from https://raspberrypi.com/software.
+Raspberry Pi Imager provides a GUI for downloading the latest version of 
+this rescue image and flashing it to an SD card.
 
 Alternatively, copy the contents of this zip file to a blank
-FAT formatted SD-CARD. The FAT partition must be < 32 GB.
+FAT formatted SD card. The FAT partition must be < 32 GB.
 
-To update the EEPROMs:
```

Comment:
There's more than one EEPROM on the intialise Pi4B

Reply:
I know. The page does not explain this thought - it mentions the utility will update the VL805 firmware, but it does not explain that this lives on a separate EEPROM chip on Pi 4B v1.1 and 1.2. I will add an explanation of this.

Reply:
Fixed.